### PR TITLE
Expenses in closed companies

### PR DESCRIPTION
**Pull request by anaschwendler**
I'll thank @jtemporal later, but by now its only missing the date comparision. ❤️

### CODE REVIEW

File: develop/2017-02-20-anaschwendler-expenses-in-closed-companies.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                             'subquota_number': np.str},
+                      low_memory=False)
+
+
+# Setting companies situation_date and reimbursements issue_date columns to correct date format, and set the cpnj to a format without dash and dots.
+
+# In[3]:
+
+reimbursements['issue_date'] = pd.to_datetime(reimbursements['issue_date'], errors='coerce')
+reimbursements['issue_date'] = reimbursements['issue_date'].apply(lambda date: date.date())
```
Comment (Irio) [inline develop/2017-02-20-anaschwendler-expenses-in-closed-companies.py]: If you remove this line and the number 35 the `query` will work as you expect.

**Comment by Irio** [comment]
Noice 👌

**Comment by anaschwendler** [comment]
Closes #194.

**Comment by anaschwendler** [comment]
Close #194

**Pull request merged**